category-committed.php

```diff
@@ -0,0 +1,330 @@
+      </div>
+    </div>
+    <div class="flex-grid-row mt-4 mb-5">
+      <div class="flex-grid-item flex-offset-s-0 flex-item-s-12 flex-offset-l-1 flex-item-l-10 flex-offset-xxl-2 flex-item-xxl-8 font-serif committed__serif-large text-paragraph-breaks">
+        <p>Britain is locking up peaceful climate protesters. You’ve probably heard of some of them – they’re the road-blockers, soup-throwers and spray-painters who’ve made headlines around the world with Just Stop Oil.</p>
+        <p>In a new podcast series from Novara Media, we meet the real people behind the front pages. What type of person would risk their freedom for a cause they believe in? What’s it really like in Britain’s overcrowded prisons – and is life on the inside any different for political prisoners? How does jail-time strain the relationships with those they love most? And when you’re facing many years inside, how do you make sense of what you’ve done?</P>
+        <p>Hosted by Rivkah Brown and Clare Hymer, Committed is a podcast about why some people are willing to go to prison for their politics.</p>
+        <p>Over four episodes, we follow a group of Just Stop Oil activists into jail and out again, asking them why they did it – and asking ourselves if we’d be willing to do the same.</p>
+      </div>
+    </div>
+    <div class="flex-grid-row mb-6 mb-s-4">
+      <div class="flex-grid-item flex-item-s-12 flex-item-xxl-12 font-color-white committed-archive-listen__box">
+        <div class="committed-archive__box">
+          <div class="committed-listen-now-title mb-1"></div>
+          <div class="font-weight-bold committed-listen-now__links font-size-13 font-size-s-12">
+            <a href="https://podcasts.apple.com/us/channel/novara-media/id6742787656">Apple Podcasts</a>,
+            <a href="https://open.spotify.com/show/3KtmyPhvQ2FvhwdRNo5I1N?si=a95692ce1d75432d">Spotify</a>,<br/><a href="#">Google Podcasts</a>,
+            <a href="https://feeds.podcastmirror.com/novara-media">RSS</a>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section id="posts" class="container">
+    <div class="mb-6">
+<?php
+if ( have_posts() ) {
+  while ( have_posts() ) {
+    the_post();
+    $meta = get_post_meta( $post->ID );
+    ?>
+      <article class="flex-grid-row pt-6 pt-s-4" id="<?php echo $post->post_name; ?>">
+        <div class="flex-grid-item flex-offset-s-0 flex-item-s-12 flex-offset-xxl-1 flex-item-xxl-4 mobile-mb-4">
+          <h4 class="font-size-9 text-uppercase font-weight-bold mb-2 mb-s-0"><?php echo $meta['_cmb_standfirst'][0]; ?></h4>
+          <h3 class="font-size-13 font-weight-semibold js-fix-widows"><?php the_title(); ?></h3>
```

Comment:
I've mostly given up on fix widows function now due to a few issues with it so can be removed.